### PR TITLE
Add sequential focus article

### PR DESCRIPTION
**Pull request by robdodson**
Waiting on https://github.com/GoogleChrome/samples/pull/315 to be merged before this can be published.

### CODE REVIEW

**Comment by robdodson** [comment]
This has already been reviewed by eng and tech writer

**Comment by jeffposnick** [comment]
It's been merged.

File: src/content/en/updates/posts/2016/03/focus-start-point.markdown [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+Keyboard handling revolves around focus, which determines where keyboard events will go in the page. There are a few situations in which, up till now, we've needed to do some extra work to make things work well for keyboard users. The `focus()` method allows us to manage focus by selectively choosing an element to focus in response to a user action. However, this best practice suffers from a lot of gotchas and requires some tricky JavaScript hackery to provide a baseline experience.
+
+While this technique isn't going to completely go away any time soon, in Chrome 50 it will be necessary in fewer cases thanks to the Sequential Focus Navigation Start Point. With this change, well-authored pages will automatically become more accessible without any need for extra manual focus management. Let's look at  an example.
+
+## Linking Within a Page
+Text heavy sites often interlink within the same page to help users quickly jump to important sections.
+
+{% highlight html %}
+<!-- Table of Contents -->
+<a href=”#recipes”>Recipes</a>
```
Comment (jeffposnick) [inline src/content/en/updates/posts/2016/03/focus-start-point.markdown]: You're using ” here and throughout, which aren't valid for quoting HTML attributes.

**Comment by robdodson** [comment]
Thanks for the quotes fixup @petele! I guess Docs inserted those :P

**Pull request merged**